skimage/future/graph/graph_cut.py

```diff
@@ -195,10 +195,17 @@ def get_min_ncut(ev, d, w, num_cuts):
         The value of the minimum ncut.
     """
     mcut = np.inf
+    mn = ev.min()
+    mx = ev.max()
+
+    # If all values in `ev` are equal, the loop does not assign a value to
+    # `min_mask`. This implies that the graph can't be further sub-divided
+    # In this case the bi-partition is the the graph itself and an empty set.
+    min_mask = np.zeros_like(ev, dtype=np.bool)
```

Comment:
I don't quite understand how this comment relates to what happens... I get `np.linspace(1, 1, 2, False)` to be `array([1., 1.])`. So, the loop below still executes. Isn't it better to shortcut the loop?